oeps/processes/oep-0056-architectural-advisory-process.rst

```diff
@@ -0,0 +1,197 @@
+======================================
+OEP-56: Architectural Advisory Process
+======================================
+
+.. list-table::
+   :widths: 25 75
+
+   * - OEP
+     - :doc:`OEP-56 <oep-0056-architectural-advisory-process>`
```

Comment:
nit: According to the OEP spec, you should be putting `proc` in the document name (`oep-0055-proc-project-maintainers`, see https://github.com/openedx/open-edx-proposals/tree/master/oeps/processes - all (except oep-1, and oep-54 where I also missed the notice) and https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0001.html#oep-header-preamble)

Reply:
👍🏻 👍🏻 👍🏻  Keep the nits coming! 😉